adserver/templates/adserver/publisher/settings.html

```diff
@@ -23,6 +23,18 @@ <h1>{% blocktrans %}Publisher settings for {{ publisher }}{% endblocktrans %}</h
     <div class="row">
 
       <div class="col-md-8 col">
+
+        {% if publisher.allowed_domains %}
+          <fieldset class="my-3 ">
+            <legend>{% trans 'Domains' %}</legend>
+            <p class="form-text">{{ publisher.allowed_domains }}</p>
```

Comment:
This looks a little odd, given it's a list displayed as a text field? Should we show it as a `ul`?

Reply:
I switched it to a UL. This is what it looks like:
![Screenshot from 2023-06-07 17-54-45](https://github.com/readthedocs/ethical-ad-server/assets/185043/a3ba628e-25bc-4bf1-bd4e-e481dd2c52d0)


I don't feel strongly one way or the other, but happy to go with this or revert the change.

Reply:
I think this looks more obviously like a list 💯